src/components/Panel.ts

```diff
@@ -299,26 +299,32 @@ class Panel implements FlickingPanel {
   public getBbox(): BoundingBox {
     const state = this.state;
     const viewport = this.viewport;
+    const element = this.element;
 
-    if (!this.element) {
+    if (!element) {
       state.cachedBbox = {
         x: 0,
         y: 0,
         width: 0,
         height: 0,
       };
     } else if (!state.cachedBbox) {
-      if (!this.element.parentNode && !viewport.options.renderExternal) {
-        viewport.getCameraElement().appendChild(this.element);
+      const wasVisible = Boolean(element.parentNode);
```

Comment:
@WoodNeck It's very trivial question. I think `isVisible` is more proper because it has parentNode currently. Why do you use `was`?

Reply:
Well, as it inserts a panel element when it wasn't visible, and as I should find panel elements that I've inserted at that sequence to remove it (that is, "wasn't visible"), I thought `wasVisible` is more proper.